### PR TITLE
Add templates for 403, 404, 500 views

### LINKED ISSUE #116: Create custom 403, 404 and 500 pages

### PR DESCRIPTION
**Pull request by noisecapella**
#### What are the relevant tickets?
Fixes #116 

#### What's this PR do?
Adds empty templates for 403, 404 and 500 views

#### How should this be manually tested?
Choose a view (for example `ui_login`) and raise one of these three exceptions: `PermissionDenied`, `NotFound`, `KeyError` for 403, 404 and 500 respectively. Set `DEBUG = False` in your settings. Verify that the appropriate template shows up and has the right status code in the network tab of your browser

### CODE REVIEW

File: odl_video/urls.py [hunk 1]
```diff
@@ -24,3 +24,7 @@
     url(r'^', include('cloudsync.urls')),
     url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework')),
 ]
+
+handler403 = 'ui.views.permission_denied_view'
+handler404 = 'ui.views.page_not_found_view'
+handler500 = 'ui.views.error_view'
```
Comment (giocalitri) [inline odl_video/urls.py]: nit: Please rename the views to contain the error code

**Comment by noisecapella** [comment]
@roberthouse54 requested that the error page show the navbar so this will need a little bit of React plumbing

File: odl_video/urls.py (hunk 1 above)
Comment (noisecapella) [reply odl_video/urls.py]: sure

**Comment by roberthouse54** [comment]
image

**Comment by noisecapella** [comment]
@pdpinch Probably a good idea, at least a minimal version of this

**Comment by noisecapella** [comment]
Should be ready for review again

**Comment by noisecapella** [comment]
I added a test for the 404 view

**Comment by giocalitri** [comment]
👍

**Pull request merged**